### ISSUE TITLE
New calendar appears only after relink (Google)

### ISSUE THREAD

**Issue by Gameniti** [marcoancona/MineTime#204]
Yesterday I created a new calendar on my google account. The standard Calendar App on my Mac got it shortly after, but Minetime (on Windows, don't know about other platforms) did not get it automatically. This new calendar only appeared after I relinked Minetime with my Google Account. I also tried "reset data" but that didn't work. (Besides that I don't really know what reset data should do since it does not seem to do anything)

Currently, I am using version 1.8.3 but I am sure that it is not a problem of this version because I installed the update an hour ago or so.

**Comment by marcoancona** [comment]
Yes, this is a limitation right now. Adding new calendars is not a very frequent operation and MineTime requires a relink to pick them up.
